app/controllers/ticket.controller.js

```diff
@@ -0,0 +1,26 @@
+var verification = require('../utils/verification.util');
+var Ticket = require('mongoose').model('Ticket');
+
+exports.createTicket = function(req, res) {
```

Comment:
This function needs to authenticate that a user is logged in to use this feature. Take a look at the listings controller to see how to do it.